### DISCUSSION TITLE
We do not want "Referral Program", becuase...

### DISCUSSION THREAD

**Discussion by IHCMATH** [IBAX-io/go-ibax#1038] [Ideas]
Almost all projects have airdrops or rewards with a reference system. This situation is beneficial for people who have a lot of followers, namely phenomena. They are given free money for themselves. What is left to your followers? Nothing... 
However, if everyone contributes to the project without saying more or less, an award should be given. This can be a kind of feedback, ease of use, developing ideas for the future goals of the project, ideas that can be further multiplied, such as social media posts. Thus, those who contribute and think are rewarded. 
There is no easy money in the world anymore. What do you think?

(Türkçe olarak da eklemek istedim yukarıdaki metni)
Hemen tüm projelerde referans sistemi ile airdrop veya ödüllendirme var. Bu durum takipçisi çok olan kişilere yani fenomenlere yarıyor. Kendileri için bedava para veriliyor. Takipçilerine ne kalıyor? Hiçbirşey... 
Halbuki herkes az veya çok demeden projeye katkı sunarsa ödül verilmeli. Bu bir çeşit geri dönüş, kullanım kolaylığı, projenin ilerideki hedeflerine yönelik fikir geliştirme, sosyal medya paylaşımları gibi daha da çoğaltılabilecek fikirler olabilir. Böylece katkı sunanlar, düşünenler ödüllendirilmiş olur. 
Artık kolay para yok dünyada. Siz ne dersiniz?

**Comment by batuhan03** [comment]
evet bi şekilde kendine topluluk yaratmak sosyal çevreni genişletmek zorundasın aslında :)
